build/Dockerfile.nginxplus

```diff
@@ -0,0 +1,34 @@
+# syntax=docker/dockerfile:1.4
+FROM alpine:3.18
```

Comment:
This will be an interesting one to track from the dependabot perspective, since N+ may not support it.

Reply:
Yeah, we'll have to cross reference with https://docs.nginx.com/nginx/technical-specs/